src/main/java/com/shapesecurity/functional/data/NonEmptyImmutableList.java

```diff
@@ -358,6 +358,18 @@ public boolean contains(@Nonnull T a) {
         return from(result);
     }
 
+    @Override
+    public boolean all(@Nonnull F<T, Boolean> f) {
```

Comment:
I would prefer this be located next to `exists`, and ideally the two should be written the same way.

Reply:
Also, if we're going to have both `exists` and `any`, we should have `every` in addition to `all`.

Or we could just have `exists` and `every`.

Reply:
I think I prefer just having `exists` and `every`. Aliases seem bad.

Reply:
Renamed to `every`, `all` deleted.